### PR TITLE
GoogleCalendarTimesDAO to fetch the auto generated meeting times

### PR DESCRIPTION
**Pull request by anna-giang**
Data Access Object to fetch the automatically generated meeting times at the 'gcal-find-times' endpoint, via the findTimes() function.

### CODE REVIEW

**Review by theresenieva — APPROVED** [review]
Looks good :)

File: GoMeet/src/main/webapp/js/google-calendar-times-dao.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    // only valid datetime strings can be converted into a Date
+    let startDate = new Date(periodStart);
+    let endDate = new Date(periodEnd);
+    if (!(startDate instanceof Date) || isNaN(startDate) ||
+        !(endDate instanceof Date) || isNaN(endDate)) {
+      throw new Error(INVALID_PARAM_VALUE);
+    }
+
+    // Fetch the relevant data from sessionStorage
+    // TODO: Refactor the getter functions that retrieve from sessionStorage
```
Comment (theresenieva) [inline GoMeet/src/main/webapp/js/google-calendar-times-dao.js:45]: I think that's a good TODO. It would be great if you had the time to do this haha.

Comment (anna-giang) [reply GoMeet/src/main/webapp/js/google-calendar-times-dao.js:45]: Yes, wish we had more time 🥺

**Review by thanhdng — APPROVED** [review]
LGTM

**Pull request merged**